testsuite/features/build_validation/retail/proxy_traditional_branch_network.feature

```diff
@@ -130,6 +131,7 @@ Feature: Prepare the branch server for PXE booting
     And I enter "proxy" in first A name field of example.org zone
     And I enter the local IP address of "proxy" in first A address field of example.org zone
     And I press "Add Item" in A section of example.org zone
+    # TODO: don't hardcode anymore the names in the private network once we have them in .bashrc
```

Comment:
Leftover or this still pending?
Have we a card to address and link this dependency?

Reply:
Yes, it's rather long term and I don't have a card. It would need a second sumaform refactor in order to pass these values in bashrc.

What do you think? We say YAGNI and I remove this remark?

Reply:
Fair enough. Let's do a card.